pkg/portal/cluster/clusteroperators.go

```diff
@@ -10,9 +10,18 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type OperatorCondition struct {
```

Comment:
It would be good if we could document the type, and what it achieves.

Reply:
Done